dimod/core/sampler.py

```diff
@@ -238,3 +239,22 @@ def sample_qubo(self, Q, **parameters):
         """
         bqm = BinaryQuadraticModel.from_qubo(Q)
         return self.sample(bqm, **parameters)
+
+    def remove_unknown_kwargs(self, **kwargs):
```

Comment:
I guess this has already been merged but I would say that this also needs to be documented somewhere in the "creating new samplers" documentation. Especially if we're expecting all samplers to do it. An example would be https://github.com/dwavesystems/dimod/pull/796/files#diff-4f1806f800c56e578e5ede71ae3a6c117743302bebb7aeddbe29404040ad052bR85 but probably it needs to be noted elsewhere.